device/mellanox/x86_64-mlnx_msn4600c-r0/Mellanox-SN4600C-D112C8/port_config.ini

```diff
@@ -0,0 +1,105 @@
+# name         lanes              alias     index    speed     fec
+Ethernet0      0,1                etp1a     0        50000     none
```

Comment:
Please change the index starting from 1, as https://github.com/Azure/sonic-buildimage/pull/4781 has been merged.

Reply:
Done